pkg/config/profile.go

```diff
@@ -139,7 +139,8 @@ func (p *Profile) GetAPIKey(livemode bool) (string, error) {
 			key = viper.GetString(p.GetConfigField(TestModeAPIKeyName))
 		}
 	} else {
-		key, err = p.RetrieveLivemodeValue(LiveModeAPIKeyName)
+		p.redactAllLivemodeValues()
```

Comment:
Would the values already be redacted by this point? Is this call defensive?

Reply:
yes its a defensive check in case user manually edits the config tom file again